bin/crew

```diff
@@ -203,8 +204,14 @@ def cache_build
       FileUtils.mv build_cachefile, "#{build_cachefile}.bak", force: true if File.file?(build_cachefile)
       FileUtils.mv "#{build_cachefile}.sha256", "#{build_cachefile}.sha256.bak", force: true if File.file?("#{build_cachefile}.sha256")
       Dir.chdir(CREW_BREW_DIR) do
-        system "tar c#{@verbose} #{pkg_build_dirname} \
-        | nice -n 20 zstd -c  --ultra --fast -f -o #{build_cachefile} -"
+        @build_cachefile_lockfile = CrewLockfile.new "#{build_cachefile}.lock"
+        begin
+          @build_cachefile_lockfile.lock
```

Comment:
Does this line do anything?  AFAICT, it only outputs the lock file.  Should it be preceded with a command like to check if it exists?

Reply:
EDIT: OK, I see this is calling the lock method of the CrewLockfile class.

Reply:
I think `../lib/crew_lockfile` handles that automatically:
```ruby
      begin
        new(path, opts).lock
      rescue LockError
        raise Errno::EEXIST, path
      end
      File.open(path, *options, &)
```